google-cloud-spanner/src/main/java/com/google/cloud/spanner/Mutation.java

```diff
@@ -402,20 +404,41 @@ private boolean isFloat32NaN(Value value) {
     return value.getType().equals(Type.float32()) && Float.isNaN(value.getFloat32());
   }
 
-  static void toProto(Iterable<Mutation> mutations, List<com.google.spanner.v1.Mutation> out) {
+  // Converts Mutation and returns a random Mutation from the available mutation list based on the
```

Comment:
```suggestion
  // Converts the list of mutations to the corresponding protobuf mutations and returns a random mutation from the available mutation list based on the
```

Reply:
Also: Method comments should be Javadoc, so change this to something like this:

```
/**
 * Converts the list of mutations to ...
*/
```

Reply:
Done